samples/other/identity-gaiax-verifier/src/main/java/org/eclipse/dataspaceconnector/iam/did/credentials/GaiaXCredentialsVerifierExtension.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.iam.did.credentials;
+
+import org.eclipse.dataspaceconnector.iam.did.spi.credentials.CredentialsVerifier;
+import org.eclipse.dataspaceconnector.iam.did.spi.hub.IdentityHubClient;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+
+import java.util.Set;
+
+/**
+ *
+ */
+public class GaiaXCredentialsVerifierExtension implements ServiceExtension {
+
+    @Override
+    public Set<String> provides() {
+        return Set.of("identity-did-verifier");
```

Comment:
That is something we can discuss at a broader scale, because ATM all extensions use non-classname-names. Should we decide to change it, we should open another issue for that. Can we defer for the moment?

Reply:
I vote for open a separate issue to define & fix the naming patterns.

Reply:
created an issues https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/79